python/cugraph_service/cugraph_service_server/cugraph_handler.py

```diff
@@ -744,7 +840,12 @@ def get_num_edges(self, edge_type, graph_id):
             else:
                 return G.get_num_edges(type=edge_type)
 
-        raise CugraphServiceError("Graph does not contain properties")
+        else:
+            if edge_type == "":
+                return G.number_of_edges()
+            else:
+                mask = G.edgelist.edgelist_df[G.edgeTypeCol] == edge_type
+                return G.edgelist.edgelist_df[mask].count()
```

Comment:
similar question here: should this code be moved into a `G.get_num_edges()` method?

Reply:
#2899

Reply:
I'll get this done right before burndown; I just have too many pots on the stove right now.